PersonApi/V1/Domain/Tenure.cs

```diff
@@ -17,5 +17,7 @@ public class Tenure
         public string Type { get; set; }
 
         public string Uprn { get; set; }
+
+        public bool IsActive => EndDate == null || DateTime.Parse(EndDate) > DateTime.UtcNow;
```

Comment:
I would not even have this logic - we are not sure what the true logic will be, maybe just return the default `false` for the time being?